docs/about/release-notes.rst

```diff
@@ -59,6 +59,7 @@ Bugfixes
 * Fix a bug in the argument detection when using setitem in combination with value-based slicing `#2750 <https://github.com/scipp/scipp/pull/2750>`_.
 * Fixed broken profile plot on 3-D data `#2746 <https://github.com/scipp/scipp/pull/2746>`_.
 * ``transform_coords`` now works with datasets with only coordinates but no data `#2755 <https://github.com/scipp/scipp/pull/2755>`_.
+* Fixed a bug when using `hist` on a Dataset that contains binned data `#2764 <https://github.com/scipp/scipp/pull/2764>`_.
```

Comment:
Very unspecific. How about ":func:`scipphist` now works with datasets that contain binned data"?